ui/src/app/views/workspace-share/component.html

```diff
@@ -1,49 +1,52 @@
-<div *ngIf="!notFound else notFoundMessage">
-  <a class="return-button" (click)="navigateBack()">&lt;&lt; Back to the workspace</a>
-  <div *ngIf="!loadingWorkspace else loading">
-    <h3>Share {{workspace.name}}</h3>
+<clr-modal [(clrModalOpen)]="sharing" [clrModalSize]="'lg'">
+  <div class="modal-title">
+    <h3 class="modal-title-text">Share {{workspace.name}}</h3>
+  </div>
+  <div class="modal-body" id="sharing-modal-body">
     <p class="paragraph-text">Workspaces can only be shared with existing All of Us users. If you want to
-    share your workspace with someone who is not yet a part of the All of Us
-    community, have them <a routerLink="">create an account</a> first and then you can share the
-    workspace with them.</p>
-   <div class="sharing-area">
-     <div class="adding-area">
-        <h4>Add a New Collaborator</h4>
-
+      share your workspace with someone who is not yet a part of the All of Us
+      community, have them <a routerLink="/" target="_blank">create an account</a> first and then you can share the
+      workspace with them.</p>
+    <div class="sharing-area">
+      <div class="adding-area">
         <div class="add-by-text-area">
-          <label class="text-label" class="find-collaborator-label">Find Collaborator</label>
+          <label class="text-label" class="collaborator-label">Find Collaborators</label>
           <input type="text" class="input share" name="to-share" autofocus
-              [(ngModel)]="toShare" placeholder="Enter username" (ngModelChange)="userNotFound = false"
+              [(ngModel)]="toShare" placeholder="enter name or {{gsuiteDomain}} ID" (ngModelChange)="userNotFound = false"
```

Comment:
"name or": Is this now implying that you can add a collaborator by name or email? Does the UI or API code support that (I had thought not)?

Personally I would probably prefer what we had previously, as it made it unambiguous as to whether you needed to include the email domain.

Reply:
this is following the comp here: https://projects.invisionapp.com/share/Q2EVSLW7Y#/screens/275889955

Reply:
"enter name" is implying functionality that we don't have, so we shouldn't include that text yet. I think the goal here should be to make this page match the mocks to the degree that we can comfortably achieve the functionality.

On the second point, I'm fine leaving the domain out as you have it here per the mocks.

Reply:
done